src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -61,15 +61,15 @@ public void TestRootGetProcessById()
             Assert.Equal(1, p.Id);
         }
 
-        [Fact]
-        public void TestUseShellExecute_Unix_Succeeds()
-        {
-            using (var p = Process.Start(new ProcessStartInfo { UseShellExecute = true, FileName = "exit", Arguments = "42" }))
-            {
-                Assert.True(p.WaitForExit(WaitInMS));
-                Assert.Equal(42, p.ExitCode);
-            }
-        }
+        //[Fact]
```

Comment:
Since you commented out tests -- are you looking for feedback on the approach, before you write tests?

Reply:
I have the failing tests commented out for now, since I'm toggling between different os as I make fixes (win, osx and linux). I will keep them commented out until all get fixes.

actually some of them are failing because I don't know if it is ok to change current behavior of Process.Start just yet or not. I'll explain in more detail in a bit when I have a good summary to share.